Standup command crashed bot

This standup command was breaking the bot. The bot would delete the message and crash before being able to resend it back formatted.

```
^standup Fixed the hibernate issues. Started prep for Cassandra. Didn't work on nodebb, discussed it with the community tho || Setup the VM for playing with cassandra. Look into plugin for nodebb. Finish atleast 3 modules in Azure dev challenge.
```

The errors in the logs was

```
/root/EddieBot/dist/timezone.js:43
        const proposedZone = TIMEZONES.find((item) => item.abbr === zone.toUpperCase());
                                                                         ^

TypeError: Cannot read property 'toUpperCase' of undefined
    at /root/EddieBot/dist/timezone.js:43:74
    at Array.find (<anonymous>)
    at Object.exports.timezone (/root/EddieBot/dist/timezone.js:43:40)
    at Client.<anonymous> (/root/EddieBot/dist/index.js:35:55)
    at Client.emit (node:events:391:22)
    at MessageCreateAction.handle (/root/EddieBot/node_modules/discord.js/src/client/actions/MessageCreate.js:31:14)
    at Object.module.exports [as MESSAGE_CREATE] (/root/EddieBot/node_modules/discord.js/src/client/websocket/handlers/MESSAGE_CREATE.js:4:32)
    at WebSocketManager.handlePacket (/root/EddieBot/node_modules/discord.js/src/client/websocket/WebSocketManager.js:384:31)
    at WebSocketShard.onPacket (/root/EddieBot/node_modules/discord.js/src/client/websocket/WebSocketShard.js:444:22)
    at WebSocketShard.onMessage (/root/EddieBot/node_modules/discord.js/src/client/websocket/WebSocketShard.js:301:10)
npm ERR! code 1
npm ERR! path /root/EddieBot
npm ERR! command failed
npm ERR! command sh -c npm run build && node dist/index.js
```

--- a/src/timezone.ts
+++ b/src/timezone.ts
@@ -24,6 +24,10 @@ export const timezone = async (message: Message) => {
   if (match) {
     const [mentioned, hour, minutes, dayNight, zone] = match;
 
+    if (!zone) {
+      return;
+    }
+
     const proposedZone = TIMEZONES.find(
       (item) => item.abbr === zone.toUpperCase()
     );

